packages/graphql/jest.global-setup.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const setTZ = require("set-tz");
+const neo4j = require("neo4j-driver");
+
+const TZ = "Etc/UTC";
+const INT_TEST_DB_NAME = "neo4jgraphqlinttestdatabase";
+
+module.exports = async function globalSetup() {
+    process.env.NODE_ENV = "test";
+
+    setTZ(TZ);
+
+    global.INT_TEST_DB_NAME = INT_TEST_DB_NAME;
+
+    const { NEO_USER = "admin", NEO_PASSWORD = "password", NEO_URL = "neo4j://localhost:7687/neo4j" } = process.env;
+    const auth = neo4j.auth.basic(NEO_USER, NEO_PASSWORD);
+    const driver = neo4j.driver(NEO_URL, auth);
+    const cypherCreateDb = `CREATE OR REPLACE DATABASE ${INT_TEST_DB_NAME}`;
+    const cypherDetachNodes = `MATCH (n) DETACH DELETE n`;
+    let session = null;
+    let cypher = null;
+
+    try {
+        const hasMultiDbSupport = await driver.supportsMultiDb();
+        session = driver.session();
+        cypher = hasMultiDbSupport ? cypherCreateDb : cypherDetachNodes;
+        await session.run(cypher);
+    } catch (error) {
+        if (error.message.includes("Unsupported administration command")) {
```

Comment:
There are many error messages that this could be as per https://github.com/neo4j/graphql/blob/dev/packages/graphql/tests/utils/is-multi-db-unsupported-error.ts - maybe worth including them?